base/pkg/entry.jl

```diff
@@ -158,15 +158,14 @@ status(io::IO, pkg::AbstractString) = status(io, pkgname = pkg)
 function status(io::IO, pkg::AbstractString, ver::VersionNumber, fix::Bool)
     @printf io " - %-29s " pkg
     fix || return println(io,ver)
-    @printf io "%-19s" ver
+    @printf io "%-9s" ver
     if ispath(pkg,".git")
         prepo = GitRepo(pkg)
         try
+            print(io, string(LibGit2.head_oid(prepo))[1:8], "  ")
```

Comment:
I would presume so, but I'm not too knowledgeable about libgit2. Does
anyone know? @wildart?

On Tuesday, July 26, 2016, Tony Kelman <notifications@github.com> wrote:

> In base/pkg/entry.jl
> <https://github.com/JuliaLang/julia/pull/17614#discussion_r72189376>:
>
> >      if ispath(pkg,".git")
> >          prepo = GitRepo(pkg)
> >          try
> > +            print(io, string(LibGit2.head_oid(prepo))[1:8], "  ")
>
> does head_oid handle the finalization that it might not be getting now
> outside of the with block?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/JuliaLang/julia/pull/17614/files/8e4b4863dc06e5977cba53668ba7bb5697e88dba#r72189376>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA492lSSF_Q5bDkXg6QUfYnCa8l3zcj5ks5qZZk9gaJpZM4JUVqo>
> .
>